modules/generic_worker/manifests/init.pp

```diff
@@ -4,13 +4,16 @@
 
 class generic_worker {
     include packages::mozilla::generic_worker
+    include generic_worker::control_bug
 
     case $::operatingsystem {
         Darwin: {
             $taskcluster_host = 'taskcluster'
             $macos_version = regsubst($::macosx_productversion_major, '\.', '')
             $taskcluster_client_id = secret('generic_worker_macosx_client_id')
```

Comment:
Don't we have different client id for staging/production?

Reply:
This change will be part of separate bug